src/networkremote/remoteclient.cpp

```diff
@@ -43,8 +40,11 @@ RemoteClient::RemoteClient(Application* app, QTcpSocket* client)
   use_auth_code_ = s.value("use_auth_code", false).toBool();
   auth_code_ = s.value("auth_code", 0).toInt();
   allow_downloads_ = s.value("allow_downloads", false).toBool();
-
+  files_root_folder_ = s.value("files_root_folder", "").toString();
+  music_extensions_ = s.value("music_extensions", "").toString();
```

Comment:
If this is really a `QStringList` I think you can store that directly in `QSettings`.

Reply:
it's easier indeed to have it as a QStringList at least at the bottom end for this test:
` if (fi.isDir() || music_extensions_.contains(fi.suffix()))`

I don't see the real benefit to store it as a QStringList in QSettings. In the GUI (network remote settings page) the User WILL define it as a coma separated list.. it sounds easier to me to just break it down in a List when needed.
i.e: in `OutgoingDataCreator::SetMusicEextensions(const QString& music_extensions)`
what you think?
I'll even add a new member: OutgoingDataCreator::music_extensions_str_ which will be an std::string to avoid doing this conversion: `files->set_music_extensions(music_extensions_.join(",").toStdString());`